### DISCUSSION TITLE
[BUG] 500 internal server error when accessing paperless from Chrome on Android

### DISCUSSION THREAD

**Discussion by landcraft** [jonaswinkler/paperless-ng#904] [Support] [answered]
**Describe the bug**
I have paperless-ng setup behind nginx (swag) and I cannot reach it from my android phone using Chrome. On my desktop, everything works fine.

**To Reproduce**
1. Go to paperless-ng URL (paperless.mydomain.com) from Chrome on Android
2. See 500 internal server error from nginx

**Expected behavior**
The front page of paperless-ng should load
**Screenshots**

**Webserver logs**
no errors in paperless.log
nginx error.log:
``` 
auth request unexpected status: 302 while sending to client, client: ClientIP, server: paperless.*, request: "GET / HTTP/2.0", host: "paperless.mydomain.com"
```

**Relevant information**
 - Host OS of the machine running paperless: [e.g. Archlinux / Ubuntu 20.04] Synology Linux OS
 - Browser [e.g. chrome, safari] Chrome on Android
 - Version [e.g. 1.0.0] 1.4.0
 - Installation method: [docker / bare metal] docker
 - Any configuration changes you made in `docker-compose.yml`, `docker-compose.env` or `paperless.conf`. no change

**Comment by landcraft** [comment, marked as answer]
sorry but I'm confused - why is this issue closed?

**Reply by shamoon** [reply]
It was moved to a discussion. Error 500s are usually a configuration / setup issue somewhere not necessarily a true bug. From your description Im guessing something with your reverse proxy or perhaps even the device itself. If it works on your desktop its not likely a problem with the project itself, no?

**Reply by landcraft** [reply]
understood. any tips on how to uncover the issue? I'm really at a loss...

**Reply by axlgit** [reply]
Hi @landcraft, I would debug nginx, What do its logs say? Of course this is off topic on paperless-ng discussion but I would start from there.

**Reply by landcraft** [reply]
OK thanks - will try that

**Reply by landcraft** [reply]
you were right - it was an nginx config issue. thanks!

**Reply by axlgit** [reply]
Glad to hear that, mark the most useful answer here as "the answer", so people will see that your question has been answered.

**Comment by matthewmorris09** [comment]
@landcraft - Would you be willing to share your swag config for paperless-ng that you got working? I haven't been able to get this to work and would love to see what a working version looks like to see what I'm messing up.

**Reply by landcraft** [reply]
Sure, I'll dig it out when I get home

**Reply by matthewmorris09** [reply]
Thank you! I appreciate you looking into this. I wasn't sure I'd hear back since the original thread is so old.

**Comment by landcraft** [comment]
Hope it helps:
```
server {
    listen 443 ssl;
    listen [::]:443 ssl;

    server_name paperless.*;

    include /config/nginx/ssl.conf;

    client_max_body_size 0;

    location / {

        include /config/nginx/proxy.conf;
        resolver 127.0.0.11 valid=30s;
        set $upstream_app paperless;
        set $upstream_port 8000;
        set $upstream_proto http;
        proxy_pass $upstream_proto://$upstream_app:$upstream_port;
    }
}
```

**Reply by matthewmorris09** [reply]
That worked. Thank you! I think I had a little too much extra configuration based on the wiki page it has about adding in some proxy_set_headers for WebSockets.
